src/comments.md

```diff
@@ -1,5 +1,24 @@
 # Comments
 
+> **<sup>Lexer</sup>**  
+> LINE_COMMENT :  
+> &nbsp;&nbsp; `//` ~[\n\r]*  
+>  
+> BLOCK_COMMENT :  
+> &nbsp;&nbsp; `/*` (BLOCK_COMMENT | .)* `*/`  
+>  
+> OUTER_DOC_LINE_COMMENT :  
+> &nbsp;&nbsp; `//!` ~[\n\r]*  
+>  
+> OUTER_DOC_BLOCK_COMMENT :  
+> &nbsp;&nbsp; `/*!` (OUTER_DOC_BLOCK_COMMENT | .)* `*/`  
```

Comment:
In outer/inner doc block comments, any kinds of block comments `/* */` `/** */` `/*! */` can be nested.